session.go

```diff
@@ -731,6 +731,11 @@ func (b *Batch) execute(ctx context.Context, conn *Conn) *Iter {
 	return conn.executeBatch(ctx, b)
 }
 
+func (b *Batch) Exec() error {
+	iter := b.session.executeBatch(b)
+	return iter.Close()
+}
```

Comment:
I think you should add a doc here because it is meant to be used by users. Maybe use the same description as for `Batch.NewBatch()`.

Also, add doc for `Batch.Query()` too.

Reply:
Agree, done.